test/spec/modules/precisoBidAdapter_spec.js

```diff
@@ -73,6 +73,7 @@ describe('PrecisoAdapter', function () {
 
       expect(data.city).to.be.a('string');
       expect(data.geo).to.be.a('object');
+      expect(data.userId).to.be.a('object');
       // expect(data.userId).to.be.a('string');
       // expect(data.imp).to.be.a('object');
```

Comment:
probably can remove commented out code.

Also, might be good to check that userId object value is `pubcid: '27e21631-975a-4410-85f3-c08b1f1c469f'` since `null` is technically an `object`

expect(data.userId).to

```js
expect(data.userId).to.deep.equal({
      pubcid: '27e21631-975a-4410-85f3-c08b1f1c469f'
});
```